style.css

```diff
@@ -112,3 +118,204 @@ img {
   align-items: center;
   gap: 20px;
 }
+
+.projects {
+  display: grid;
+  grid-template-columns: auto;
+  justify-content: center;
+  row-gap: 20px;
+  padding: 10px 40px;
+  box-sizing: content-box;
+  font-family: 'Inter', Courier, monospace;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.project-card {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+}
+
+.projects-section-title {
+  font-family: 'Crete Round', Courier, monospace;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  margin-top: 100px;
+}
+
+.desc-title {
+  font-family: 'Crete Round', Courier, monospace;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  margin: 0;
+}
+
+.description {
+  font-family: 'Inter', Courier, monospace;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+}
+
+.project-image img {
+  width: 100%;
+  height: auto;
+}
+
+.project-tech {
+  display: flex;
+  gap: 20px;
+  justify-content: start;
+}
+
+.project-tech li a {
+  box-sizing: border-box;
+  font-size: 15px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  border: 1px solid black;
+  font-family: 'Inter', Courier, monospace;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 20px;
+  color: #091e42;
+}
+
+.project-button {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  width: 124px;
+  height: 48px;
+  background: #ff6b00;
+  font-family: 'Inter', Courier, monospace;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+  border: none;
+}
```

Comment:
- [ ] Buttons are almost there! However, it would be great to add some states (Active, Hover and Pressed) like in the figma template:

![Captura de pantalla 2022-05-27 a la(s) 10 08 42 a  m](https://user-images.githubusercontent.com/83717285/170727640-319156db-d33a-4cbd-964b-e60821bf2440.png)

Reply:
##  Add button states and fixing html semantics 
- Use list (ul) for technology categories.
- Add button states (hover, pressed and disabled);
thanks for very much for taking your time and reviewed my work, i have done all the changes according to the reviews provided i hope now you can check.